### DISCUSSION TITLE
rsync error

### DISCUSSION THREAD

**Discussion by tomektomczuk** [wpengine/github-action-wpe-site-deploy#68] [Q&A]
What you guys set as SRC_PATH, REMOTE_PATH as for me default paths create error in rsync

`rsync: [sender] change_dir "/github/workspace/wp-content/themes/MYTHEME" failed: No such file or directory (2)
...
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1336) [sender=3.2.7]`

**Comment by tezell-mg** [comment]
I'm also getting some files/attrs were not transferred.  However, don't have any errors present above that line.  rsync is exiting with the status 0.  @tomektomczuk were you able to resolve this issue on your end?

**Reply by tomektomczuk** [reply]
I managed to make this action work.

**Reply by tezell-mg** [reply]
@tomektomczuk I got it to succeed as well.  My issue was that rsync was getting a permission denied error on one of the mu-plugins.  I ignored those and removed them from the repo and everything is working now.

**Reply by MAR-GUEZ** [reply]
> I managed to make this action work.

How did you solve it?
